src/components/charts/NetWorthHistogram.tsx

```diff
@@ -21,70 +21,85 @@ C.register(
 );
 
 export type NetWorthHistogramProps = {
+  assetsGuid: string,
+  assetsLabel?: string,
+  hideAssets?: boolean,
+  liabilitiesGuid: string,
+  liabilitiesLabel?: string,
+  hideLiabilities?: boolean,
   selectedDate?: DateTime,
+  showLegend?: boolean,
```

Comment:
reconverting to a re-usable component where you can choose to show/hide some of the datasets.